src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -103,6 +103,7 @@ public void Startup(StartupParams startupParams)
                 new MutatingFileSystem(),
                 new CustomNodePathRemapper(startupParams.CustomNodeManager, DynamoModel.IsTestMode));
 
+            PackageUploadBuilder.SetEngineVersion(startupParams.DynamoVersion);
             var uploadBuilder = new PackageUploadBuilder(dirBuilder, new MutatingFileCompressor());
```

Comment:
@pboyer , this `engineVersion` is used then in static method
`public static PackageUploadRequestBody NewRequestBody`.
That's why I set it this way. 

Passing version as an argument to `PackageUploadRequestBody` method seems to me not quite good idea, because it calls from many places, but in fact, this argument is the same during whole work of the application, so passing version to the static method each time seems redundant.

What do you think about it?